new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -214,156 +224,281 @@ public void testCreateMiiDomain() {
-            domainUid, domainNamespace));
-
+    int adminNodePort = 30711;
+    createDomainResource(domainUID, domainNamespace, adminSecretName, repoSecretName,
+            encryptionSecretName, adminNodePort, replicaCount);
 
     // wait for the domain to exist
     logger.info("Check for domain custom resouce in namespace {0}", domainNamespace);
     withStandardRetryPolicy
         .conditionEvaluationListener(
             condition -> logger.info("Waiting for domain {0} to be created in namespace {1} "
                     + "(elapsed time {2}ms, remaining time {3}ms)",
-                domainUid,
+                domainUID,
                 domainNamespace,
                 condition.getElapsedTimeInMS(),
                 condition.getRemainingTimeInMS()))
-        .until(domainExists(domainUid, DOMAIN_VERSION, domainNamespace));
+        .until(domainExists(domainUID, DOMAIN_VERSION, domainNamespace));
 
 
     // check admin server pod exist
     logger.info("Check for admin server pod {0} existence in namespace {1}",
         adminServerPodName, domainNamespace);
-    checkPodCreated(adminServerPodName);
+    checkPodCreated(adminServerPodName, domainUID, domainNamespace);
 
     // check managed server pods exists
     for (int i = 1; i <= replicaCount; i++) {
       logger.info("Check for managed server pod {0} existence in namespace {1}",
           managedServerPrefix + i, domainNamespace);
-      checkPodCreated(managedServerPrefix + i);
+      checkPodCreated(managedServerPrefix + i, domainUID, domainNamespace);
     }
 
     // check admin server pod is running
     logger.info("Wait for admin server pod {0} to be ready in namespace {1}",
         adminServerPodName, domainNamespace);
-    checkPodRunning(adminServerPodName);
+    checkPodRunning(adminServerPodName, domainUID, domainNamespace);
 
     // check managed server pods are running
     for (int i = 1; i <= replicaCount; i++) {
       logger.info("Wait for managed server pod {0} to be ready in namespace {1}",
           managedServerPrefix + i, domainNamespace);
-      checkPodRunning(managedServerPrefix + i);
+      checkPodRunning(managedServerPrefix + i, domainUID, domainNamespace);
     }
 
     logger.info("Check admin service {0} is created in namespace {1}",
         adminServerPodName, domainNamespace);
-    checkServiceCreated(adminServerPodName);
+    checkServiceCreated(adminServerPodName, domainNamespace);
 
     // check managed server services created
     for (int i = 1; i <= replicaCount; i++) {
       logger.info("Check managed server service {0} is created in namespace {1}",
           managedServerPrefix + i, domainNamespace);
-      checkServiceCreated(managedServerPrefix + i);
+      checkServiceCreated(managedServerPrefix + i, domainNamespace);
+    }
+
+  }
+
+  @Test
+  @Order(2)
+  @DisplayName("Create a second domain with the image from the the first test")
+  @Slow
+  @MustNotRunInParallel
+  public void testCreateMiiSecondDomain() {
```

Comment:
Shall we indicate in the test name that the second domain is in the same namespace as the first domain?

Reply:
The domain is created in a different Namespace (as per Monica's requirement), but I will change the test name to indicate that.